src/script/components/MessagesList/Message/ContentMessage/asset/ImageAsset.test.tsx

```diff
@@ -78,10 +77,17 @@ describe('image-asset', () => {
         version: 3,
       }),
     );
-    const testPage = new ImageAssetTestPage({...defaultProps, asset: image});
-    await waitForExpect(() => {
+
+    const props = {...defaultProps, asset: image};
+
+    const {container} = render(<ImageAsset {...props} />);
+
+    const imageElement = container.querySelector('[data-uie-name="image-asset-img"]');
```

Comment:
Here you can use testing lib primitives (like `getByTestId('image-asset-img')`). 
It gives a super nice report when the thing is not found in the component. 

In general, you should, as much as possible, try to avoid using `container` as it's very low level and gives very poor error reports